service/src/main/kotlin/fi/espoo/evaka/application/ApplicationControllerV2.kt

```diff
@@ -208,7 +213,9 @@ class ApplicationControllerV2(
                 periodStart = periodStart,
                 periodEnd = periodEnd,
                 searchTerms = searchTerms ?: "",
-                transferApplications = transferApplications ?: TransferApplicationFilter.ALL
+                transferApplications = transferApplications ?: TransferApplicationFilter.ALL,
+                authorizedUnitFilterList = authorizedUnitFilterList,
```

Comment:
I think it would be better to pass an `AclAuthorization` object here to the function (obtained with `acl.getAuthorizedUnits(user)`, because it *attaches meaning* to an optional list/set of IDs. Just like we prefer passing `AuthenticatedUser` instead of just `UUID` if a function needs a user ID.

Then, in the query binding you should be able to do `aclAuthorization.ids?.toTypedArray()`, and it handles all the details automatically